ga4gh/flaskapp.py

```diff
@@ -0,0 +1,73 @@
+"""
+Contains the actual flask startup code
+"""
```

Comment:
I'm not really seeing the benefit of this new module. We still have lots of flasky stuff in frontend.py, so we're not breaking flask from everything else. I'd vote for keeping things are they are here.